sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py

```diff
@@ -1910,7 +1912,7 @@ def deserialize_date(attr):
         if re.search(r"[^\W\d_]", attr, re.I + re.U):  # type: ignore
             raise DeserializationError("Date must have only digits and -. Received: %s" % attr)
         # This must NOT use defaultmonth/defaultday. Using None ensure this raises an exception.
-        return isodate.parse_date(attr, defaultmonth=None, defaultday=None)
+        return isodate.parse_date(attr, defaultmonth=0, defaultday=0)
```

Comment:
I know this is generated code, but Im guessing this change will not impact the exception behavior from the comment?